src/grandorgue/gui/GOGUISequencerPanel.cpp

```diff
@@ -39,106 +43,152 @@ GOGUIPanel *GOGUISequencerPanel::CreateSequencerPanel(GOConfigReader &cfg) {
 
   GOGUILabel *PosDisplay
     = new GOGUILabel(panel, m_OrganController->GetLabel(wxT("Label")));
-  PosDisplay->Init(cfg, wxT("SetterCurrentPosition"), 350, 10);
+  PosDisplay->Init(cfg, wxT("SetterCurrentPosition"), 325, 10);
+  PosDisplay->SetFontSize(14);
```

Comment:
I'm perfectly ok with this, but in the long run I think we should have another way of exposing the font and font size for both label and button elements so that the user can customize them from within GO.

Reply:
Is not resizing the whole panel sufficient?

And now it is possible to change it in odf with

```
[SetterCurrentPosition]
FontSize=16
```